### ISSUE TITLE
startup_macosx.sh needs update

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1510]
My default shell is zsh, and open the GMT app gives warnings:
```
**Warning: your default shell is /bin/zsh.  
GMT module commands are only available in BASH.  
Change your default shell or make sure /Applications/GMT-6.0.0rc4.app/Contents/Resources/bin 
is in your PATH and use /Applications/GMT-6.0.0rc4.app/Contents/Resources/share/tools/gmt5syntax 
to convert your old GMT scripts.**
```

A few issues here:

1. The warning message is too long to fit a 80-character terminal.
2. GMT commands are available in any shell. The only bash-only stuff is the bash completion.
3. The message "use gmt5syntax to convert your old GMT scripts" seems useless.

**Comment by PaulWessel** [comment]
This is in cmake/dist/startup_macosx.sh.in.  Feel free to make changes.  I gotta go teach.

**Comment by seisman** [comment]
Closed by #1515
